assets/questions/i18n.js

```diff
@@ -111,9 +116,9 @@ def createListOfUniqueStrings(atom, size):
       expectedOutput: ''
     }],
     buggyOutputTests: [{
-      buggyFunction: 'useFirstAndLastLetterAndLengthToAbbreviate',
+      buggyFunction: 'AuxiliaryCode.useFirstAndLastLetterAndLengthToAbbreviate',
       messages: [
-        "It looks like your output (%s) doesn't match our expected output (%s).",
+        "Try running your code on 'at'. What does it produce?",
```

Comment:
Ditto, here and elsewhere. May not need changing, but just wanted to check what you thought.